rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1915,8 +1915,7 @@ public void steps(Program aprogram, long steps) {
                 vmCounter++;
             } // for
         } catch (RuntimeException e) {
-                logger.warn("VM halted: [{}]", e);
-                panicProcessor.panic("vm", String.format("VM halted: [%s]", e.getMessage()));
+                logger.error("VM halted: [{}]", e);
```

Comment:
Can you verify this is working correctly? I think the string formatting doesn't apply when the second parameter is of type `Exception`.

Reply:
:+1: